opsdroid/connector/sms/__init__.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+# Opsdroid Imports
+from opsdroid.connector import Connector, register_event
+from opsdroid.events import Message
+import logging
+
+# Twilio Imports
+from twilio.rest import Client
+
+_LOGGER = logging.getLogger(__name__)
+CONFIG_SCHEMA = {
+    "name": str,
+    "account_sid": str,
+    "auth_token": str,
+    "phone_number": str,
+    "host": str,
+    "port": int,
+}
```

Comment:
I assume some of these are not optional. They should have `voluptuous.Required` around them.

See the Slack connector for [an example](https://github.com/opsdroid/opsdroid/blob/af4253ea542aeec78f86eee426d56abdc937d1ec/opsdroid/connector/slack/__init__.py#L29) of this.

Reply:
@jacobtomlinson thanks for reviewing this! I have resolved this